hilda/snippets/dyld.py

```diff
@@ -1,39 +1,126 @@
-import logging
+from functools import cached_property
+
 import lldb
+from construct import *
```

Comment:
i usually use this import style only while developement and debugging, but please import each subcon from `construct` manually. PyCharm should handle it pretty good using `Alt+Enter` on each red name.